README.md

```diff
@@ -55,6 +55,33 @@ if command -v nix-your-shell > /dev/null; then
 fi
 ```
 
+### Nushell
+
+> **Note**
+> Nushell requires sourced configuration files to exist before `nu` is started.
+
+Add to your `~/.config/nushell/config.nu`:
+
+```nu
+source nix-your-shell.nu
+```
+
+To generate `nix-your-shell.nu` file:
+
+Either manually generate it:
+
+```sh
+nix-your-shell nu | save nix-your-shell.nu
+```
```

Comment:
Non-blocking: Maybe we should have a `nix-your-shell nu install` command, which generates the correct configuration and puts it in a place the shell will notice, e.g. appending it to `~/.config/nushell/config.nu`?

Reply:
we can do something like this, but I do prefer more explicit configurations instead of messing up with user configurations. I'm still not sure what the best way to provide the integrations with nushell. One of the aspect of this - the nixos + home-manager users that won't benefit from such command and for them would be better to provide integration on home-manager side.